src/main/resources/hudson/scm/cvstagging/CvsTagAction/create-another-tag.js

```diff
@@ -0,0 +1,9 @@
+document.addEventListener('DOMContentLoaded', () => {
+
+    document.getElementById('create-another-tag-button')
+        .addEventListener('click', () => {
+            document.getElementById('tagForm').show();
```

Comment:
Is `show` a method of some specific element? I tried looking it up on MDN, but nothing relevant shows up.

Reply:
Unfortunately I think the original code was Prototype.js code, broken from when we removed Prototype.js from Jenkins core several months back. :-(